app/views/step_by_step_pages/show.html.erb

```diff
@@ -60,18 +60,13 @@
             </td>
           </tr>
         <% end %>
-
       </tbody>
     </table>
-    <ul class="list-inline publish-actions">
-      <% if @step_by_step_page.has_been_published? %>
-        <li><%= link_to 'Unpublish', step_by_step_page_unpublish_path(@step_by_step_page), class: "btn btn-danger" %></li>
-      <% else %>
-        <li><%= link_to 'Delete', @step_by_step_page, method: :delete, data: { confirm: 'Delete this step by step page?' }, class: "btn btn-danger" %></li>
-      <% end %>
-      <% if @step_by_step_page.has_draft? %>
-        <li><%= link_to 'Publish', step_by_step_page_publish_path(@step_by_step_page), class: "btn btn-primary" %></li>
-      <% end %>
-    </ul>
+    <div class="col-md-2 col-md-offset-10">
+      <ul class="list-unstyled step-by-step-actions text-right">
+        <li><%= link_to 'Add a new step', new_step_by_step_page_step_path(@step_by_step_page), class: "btn btn-primary btn-block" %></li>
```

Comment:
Are these buttons being added, or just being moved on the page?

Either way, I think this commit should just focus on the movement of the "Publish", "Unpublish" and "Delete" buttons